test/Config.spec.ts

```diff
@@ -8,6 +8,7 @@ describe("Config", async () => {
   const [user1, user2, user3] = waffle.provider.getWallets();
   const crankAddress = user2.address;
   const feeReceiver = user3.address;
+  const validForDays = 259200; //3 days
```

Comment:
Ah, it's not stored as days. I think this is a bit of a footgun - I think maybe we should rename it to `validForSeconds` in the contract code

Reply:
I'd keep it in days, i.e. `validForDays = 3`, this is how we do it in the hub also

Reply:
updated